documentation/proc-pages/io/input-guide.md

```diff
@@ -51,7 +51,7 @@ which are discussed below.
 !!! Info "Constraints"  
     A full list of constraints is given on the variable description page in the row labelled 
     `lablcc` [here](../vardes/#numerics).  
```

Comment:
This link leads to a 404 error

Reply:
You need to run `vardes.py` to generate vardes.md, then all the links should work. This script would normally be run on the CI so you have to run it manually to get the variable descriptions page to work.

Reply:
Link now works as expected